src/eko/evolution_operator/__init__.py

```diff
@@ -424,15 +449,28 @@ def compute(self):
                 self.fact_to_ren,
                 "exponentiated" if self.sv_mode == 1 else "expanded",
```

Comment:
Here we should do a better job in general, and start defining `enum`s for this kind of objects.